src/leap/bitmask/backend/backend_proxy.py

```diff
@@ -28,6 +28,7 @@
 import zmq
 
 from leap.bitmask.backend.api import API, STOP_REQUEST, PING_REQUEST
+from leap.bitmask.backend.settings import Settings
```

Comment:
It looks that you're only creating the Settings object but not using anywhere, am I missing something?